js/functions.js

```diff
@@ -0,0 +1,33 @@
+const checkString = (testString, maxLength) => testString.length <= maxLength;
+
+const isPalindrome = (testString) => {
+  testString = testString.toLowerCase().replaceAll(' ', '');
```

Comment:
Есть правило не переопределять параметры в функциях, был бы линтер строже настроен он бы [ругался](https://eslint.org/docs/latest/rules/no-param-reassign)
```suggestion
  const formattedString = testString.toLowerCase().replaceAll(' ', '');
```